doc/dash/dash-sonic-hld.md

```diff
@@ -507,6 +523,8 @@ key                      = DASH_APPLIANCE_TABLE:id ; attributes specific for the
 sip                      = source ip address, to be used in encap
 vm_vni                   = VM VNI that is used for setting direction. Also used for inbound encap to VM
 local_region_id          = Region where this appliance is located
+outbound_direction_lookup= dst_mac or src_mac; Default is src_mac. This attribute overrides to dst_mac
+trusted_vni              = list of global trusted VNIs, 'comma' seperated or "-" for range both inclusive.
```

Comment:
trusted_vnis?

Reply:
IMO, we do both ways in the schema and no defined convention.

Reply:
changed to vnis